src/components/dialog-table.js

```diff
@@ -29,7 +29,7 @@ class DialogTable extends React.Component {
                 maxWidth="lg"
             >
                 <DialogTitle id="form-dialog-title">
-                    Values per {this.props.title}
```

Comment:
@Sharmyn28 this should be `i18n.t('values per {{propName}}', { propName: this.props.title })` so that it's easier to translate